packages/plans-grid-next/src/components/features-grid/features-item.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../mixins";
+
+.features-grid__features-item {
```

Comment:
What do you think about a common prefix for all the classes/elements defined under `plans-grid-next` package? I think we should aim to increase the specificity of these classes - perhaps prefixing everything with `plans-grid-next` (or `pgn-`, or something else)? It's easy otherwise to have competing styles - especially when looking at the more general classes like "comparison-grid" or "features-grid" (already the former is coming out of `@automattic/components` and those don't have a root prefix either).